CHANGELOG.unreleased.md

```diff
@@ -16,6 +16,7 @@
 - [Incremental Backup & Replication] Attempt to work around HVM multiplier issues when creating VMs on older XAPIs (PR [#6866](https://github.com/vatesfr/xen-orchestra/pull/6866))
 - [REST API] Fix VDI export when NBD is enabled
 - [XO Config Cloud Backup] Improve wording about passphrase (PR [#6938](https://github.com/vatesfr/xen-orchestra/pull/6938))
+- [Import/VM/From VMware] Improve import performance to ensure server is not overloaded (PR [#6919](https://github.com/vatesfr/xen-orchestra/pull/6919))
```

Comment:
I don't understand this message and I fear users will not either.

Reply:
maybe we don't need to put a changelog

Reply:
It's up to you :slightly_smiling_face: